server/executor/runner.go

```diff
@@ -70,6 +70,7 @@ func (r persistentRunner) Start(workers int) {
 					fmt.Println("persistentRunner exit goroutine")
 					return
 				case job := <-r.executeQueue:
+					job.run.StartedAt = time.Now()
```

Comment:
I would put this either at the `processExecQueue` func, or remove the field, since it's almost the same value as `ServiceTriggeredAt`